templates/home/home_page.html

```diff
@@ -51,6 +51,24 @@ <h2 class="tna-heading tna-heading--l">
         </div>
 
     </div>
+
+<!--graph ql banner -->
+<div class="tna-container">
+
+    <div class="tna-column tna-column--width-1-5 tna-column--width-1-3-medium tna-column--full-small tna-column--full-tiny graph-banner-icon">
+        <img src ="/static/images/graph-ql-icon.jpg" alt="graphQL icon">
+    </div>
+    <div class="tna-column tna-column--width-3-4 tna-column--width-2-3-medium tna-column--full-small tna-column--full-tiny graph-banner">
+        <h3 class="tna-heading tna-heading--l">
+            <a href="">Query the data</a>
```

Comment:
The Graph QL end point is missing. Should the link open in a new tab/window?

Reply:
We don't know the final url of the graph QL url as yet. Think it will open in the same window.